src/Agent/CHANGELOG.md

```diff
@@ -8,7 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### New Features
 * **New Feature Template** <br/>
 New Feature Description
-Fixes Issue [#XX](https://github.com/newrelic/newrelic-dotnet-agent/issues/XX)
+
+### Fixes
+Fixes Issue [#224](https://github.com/newrelic/newrelic-dotnet-agent/issues/224): leading "SET" commands will be ignored when parsing compound SQL statements.
```

Comment:
It is a good practice to also reference the PR at the end of the fix message . For example, "Fixes issue #301 where the agent incorrectly parses server-side configuration causing agent to shutdown.(#310)"

Reply:
Good idea, thanks.